_docs/2.0.0/usage/schema.md

```diff
@@ -272,7 +272,7 @@ Here are the recognized ones with the description of their purpose (with detaile
 | [minus_sign](#minus_sign)                 | any numeric type            | Character to mark the number is negative.                                                                                                             | *N*             | *-*                                      |
 | [allow_infinity](#allow_infinity)         | float & double              | Flag indicating if the column accepts infinity as a value (and positive/negative numbers which are too large are converted to *infinity*/*-infinity*) | *true*          | *false*                                  |
 | [radix](#radix)                           | long, integer, short, byte  | The base of the numbers provided                                                                                                                      | *hex*           | *10*                                     |
-| [encoding](#encoding)                     | string ("base64" or "none") | Encoding used for string->binary conversion                                                                                                           | *base64*,*none* | (no encoding present)                    |
+| [encoding](#encoding)                     | binary                      | Encoding is used for string to binary conversion                                                                                                      | *base64*,*none* | *none*                                   |
```

Comment:
*none* is strictly speaking not default here, only implied resulting in possible warnings. 🤔

Reply:
Yes. I was about to point that out, but then I realized we can hardly clarify it completely in the table row.

The description of the `encoding` metadata key-value explains this in sufficient detail, so I was hoping it should suffice https://github.com/AbsaOSS/enceladus/pull/1388/files#diff-e4c9947563dc83be794d2f43bd711113R397

I'll just make a link to it from here.